blocks/form/gdpr-confirmation.js

```diff
@@ -0,0 +1,78 @@
+// get url parameters
+const curUrl = window.location.href;
+const urlParams = new URLSearchParams(window.location.search);
+const paramdata = urlParams.get('data');
+const paraminit = urlParams.get('init');
+
+function hasQueryParams() {
+  return curUrl.toString().includes('?');
+}
+
+// Send a query parameters to the GDPR API Accept
+export async function acceptresult(accept, urlProd, secondConfig) {
+  const url = urlProd + accept;
+  const options = {
+    method: 'PUT',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      data: paramdata,
+      init: paraminit,
+      confirm: 'true',
+    }),
+  };
+  try {
+    const response = await fetch(url, options);
+    const resData = await response.text();
+    if (response.ok) {
+      window.open(secondConfig);
+    } else {
+      const { message } = JSON.parse(resData);
+      throw message;
```

Comment:
we should have a proper error handling for the enduser, as far as I can see this is only giving an output on the browser console